core/storage/config/gae_models_test.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(config_models,) = models.Registry.import_models([models.NAMES.config])
+
+
+class ConfigPropertyModelUnitTests(test_utils.GenericTestBase):
+    """Test ConfigPropertyModel class."""
+
+    def test_create_model(self):
+        config_model = config_models.ConfigPropertyModel(
+            value='b')
+        self.assertEqual(config_model.value, 'b')
+
+    def test_commit(self):
+        config_model1 = config_models.ConfigPropertyModel(
+            id='config_model1', value='c')
+        config_model1.commit(feconf.SYSTEM_COMMITTER_ID, [])
+        config_model1 = config_models.ConfigPropertyModel.get_version(
+            'config_model1', 1)
+        self.assertEqual(config_model1.value, 'c')
+        config_model1.value = 'd'
+        config_model1.commit(feconf.SYSTEM_COMMITTER_ID, [])
+        retrieve_commit = config_models.ConfigPropertyModel.get_version(
```

Comment:
It's a bit confusing to call this retrieve_commit when the one on line 36 is called config_model1.

Maybe call this config_model2 instead?

Reply:
Or you can call the retrieved commit above retrieved_model_1 and this one retrieved_model_2.

Note that it's not appropriate to describe this version as a "commit". "Commit" is a verb. This object you're retrieving is really a version of a model instance.

Reply:
I missed your third comment here initially, but I just changed it to use retrieved_model instead.